### DISCUSSION TITLE
"Rocket.Chat.Electron" and "Rocket.Chat.React Native"

### DISCUSSION THREAD

**Discussion by yoshida-git-ai** [RocketChat/Rocket.Chat.ReactNative#4131] [Q&A]
We build and operate a company internal chat system at Rocket.Chat.Electron.

Recently I have a request to do it on Android

I'm thinking of building Rocket.Chat.ReactNative.

How does the existing Rocket.Chat.Electron (PC) and Rocket.Chat.ReactNative (mobile) work together?

First of all, what I want to know is that it is possible to cooperate with any of Rocket.Chat.ReactNative settings?

Or do I need to modify the Rocket.Chat.React Native code quite a lot?

**Comment by yoshida-git-ai** [comment]
In other words

Do we need to start the server just for mobile applications?

Android studio starts on Windows and edits to create an APK.

Need to build a server such as Linux?
Or connect to Rocket.Chat.Electron's server?

I would like to know the overall overview.

**Comment by dnlsilva** [comment]
The same server you use for the desktop you can use for the react native app.
You don't need to make any changes.

You'll need to change a few things if you wanted to make your own app (with the company's identity, a whitelabel).
However, if it is not necessary, you can tell the staff to download the rocket.chat app directly from the playstore and connect to your server.

**Reply by yoshida-git-ai** [reply]
Thanks for your advice

Rocket.Chat.Electron (PC) will continue to operate.

First of all, I'm trying to build Rocket.Chat.ReactNative (mobile) by default.
Now I have successfully started Metro on Windows.
![metro](https://user-images.githubusercontent.com/94094919/166203917-b2ddad85-c658-44e3-96f9-94100a863e5d.png)

Settings for connecting to Rocket.Chat.Electron (PC)

(Include whitelist? In configuration files, etc.)
And does that mean I can work together if I build Rocket.Chat.ReactNative (mobile)?
